VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
Ok, I think I've reached out to the right people, but I kinda just sent an email to somewhere random and am hoping it works ;).